src/app/CommandSender.h

```diff
@@ -124,9 +180,49 @@ class CommandSender final : public Messaging::ExchangeDelegate
     CommandSender(Callback * apCallback, Messaging::ExchangeManager * apExchangeMgr, bool aIsTimedRequest = false,
                   bool aSuppressResponse = false);
     ~CommandSender();
-    CHIP_ERROR PrepareCommand(const CommandPathParams & aCommandPathParams, bool aStartDataStruct = true);
-    CHIP_ERROR FinishCommand(bool aEndDataStruct = true);
+
+    /**
+     * Enables additional features of CommandSender, for example sending batch commands.
+     *
+     * In the case of enabling batch commands, once set it ensures that commands contain all
+     * required data elements while building the InvokeRequestMessage such that it adhears to the
+     * spec. This must be called before PrepareCommand.
```

Comment:
"adheres".

But also, I don't understand this "ensures .. all required data elements".  Is that just saying "even if you call this method we won't start violating the spec"?  Why does that need to be called out?